bot/exts/evergreen/catify.py

```diff
@@ -0,0 +1,44 @@
+import random
+from typing import Optional
+
+from discord.ext import commands
+
+from bot.constants import Cats
+
+
+class Catify(commands.Cog):
+    """Cog for the catify command."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(aliases=["ᓚᘏᗢify", "ᓚᘏᗢ"])
+    async def catify(self, ctx: commands.Context, *string: Optional[str]) -> None:
+        """Catifies a string or username."""
+        if string == ():
```

Comment:
```suggestion
    async def catify(self, ctx: commands.Context, *, string: Optional[str]) -> None:
        """Catifies a string or username."""
        if not str:
```
This would remove the need for `" ".join`ing the string.

Reply:
This also breaks the command

Reply:
I mentioned that it would remove then need of `" ".join`ing the string.

Reply:
I tested this again, still breaks